### PR TITLE
Utiltiy function to use Hexadecimal color code

### PR DESCRIPTION
**Pull request by namanb009**
This function will allow users to use hexadecimal color throughout FURY instead of rgb.

Type: Enhancement
Improves: User usability.

### CODE REVIEW

File: fury/utils.py [hunk 1]
```diff
@@ -964,3 +964,14 @@ def check_order(vert, triarr):
         else:
             correct_vert[nb] = i
     return correct_vert
+
+def hex_color(color):
```
Comment (skoudoro) [inline fury/utils.py]: Can you rename this function `hex2rgb` or `hex_to_rgb`?
Can you add docstring (documentation for this function? Look at the functions above

File: fury/utils.py [hunk 2]
```diff
@@ -964,3 +964,14 @@ def check_order(vert, triarr):
...
+    if color[0] == "#":
+        color = color[1:]
+
+    r = int("0x" + color[0: 2], 0) / 256
+    g = int("0x" + color[2: 4], 0) / 256
+    b = int("0x" + color[4: 6], 0) / 256
+
+    return((r, g, b))
```
Comment (skoudoro) [inline fury/utils.py]: `return np.array([r, g, b])` instead. we prefer numpy array instead of tuple

File: fury/utils.py [hunk 3]
```diff
@@ -964,3 +964,14 @@ def check_order(vert, triarr):
         else:
             correct_vert[nb] = i
     return correct_vert
+
+def hex_color(color):
+    if color[0] == "#":
+        color = color[1:]
+
+    r = int("0x" + color[0: 2], 0) / 256
```
Comment (skoudoro) [inline fury/utils.py]: replace `/` by  `//` to avoid a float value. 

Same comment below.

Comment (namanb009) [reply fury/utils.py]: r, g, and b are supposed to have a float value between 0 and 1. Hence, I have used / operator

**Comment by namanb009** [comment]
@skoudoro The utils.py file in this branch just went missing, I don't know-how. So I added it again on my own and now I'm stuck
Can you help me out here?

**Comment by skoudoro** [comment]
Hi @namanb009, 

it looks like you renamed `utils.py` to `colormap.py`. What you should do is:

- go to the folder where there is `setup.py`
- git remove `colormap.py`
- cd fury
- git checkout `utils.py `
- git push

and you should be ok

**Pull request closed without merging**